data/src/test/java/org/apache/iceberg/data/TestMetricsRowGroupFilter.java

```diff
@@ -297,7 +297,9 @@ public void testNoNulls() {
   }
 
   @Test
-  public void testIsNaN() {
+  public void testIsNaNForParquet() {
```

Comment:
EDIT - I see now why the name was changed.

My suggestion, to avoid modifying the name / making two tests that are named specifically for their file format, that you consider making two separate functions and  calling out to each situationally.

Like
```java
public void testIsNaN() {
  Assume.assumeTrue("Avro files do not have row group statistics", format != FileFormat.AVRO);
  
  if (format == FileFormat.ORC) {
    testIsNanForOrc();
  } else {
    testIsNanForParquet();
  }
}

private void testIsNanForOrc() { .... }

private void testIsNanForParquet() { ... }
```

This way, the same test (by name) will be run for each, but the correct assertions will be handled.